src/SFA.DAS.EmployerAccounts.TestCommon/EmployerAgreementBuilder.cs

```diff
@@ -104,7 +105,7 @@ public EmployerAgreementBuilder WithAgreement(long accountId, long legalEntityId
                 Template = template,
                 TemplateId = template.Id,
                 StatusId = status,
-                AccountLegalEntityId = accountLegalEntityId
+                AccountLegalEntityId = accountLegalEntityId,
```

Comment:
trailing comma can be removed

Reply:
I have no idea why I randomly added a comma to a piece of code I hadn't otherwise changed!